src/com/hoffrogge/uebungen/KlassenWeitergeben.java

```diff
@@ -0,0 +1,25 @@
+package com.hoffrogge.uebungen;
+
+import java.util.ArrayList;
+
+import com.hoffrogge.lehreinheit02.Wuerfel;;
+
+public interface KlassenWeitergeben {
+	
+	public ArrayList<Wuerfel> instanceList = new ArrayList<Wuerfel>();
```

Comment:
Sobald Interfaces eingeführt werden, wollen wir natürlich auch gegen Interfcaes implementieren, daher hier bitte List<Wuerfel> instanceList = new ArrayList<Wuerfel>(); (das public kann auch entfallen, das ist durch das Interfce bereits gegeben).
Des Weiteren ist es aber auch nicht schön, eine Variable in einem Interface zu haben, das ist ein Implementierungsdetail, um das sich das Interface nicht kümmern sollte.